src/main_thread/api/public_api.ts

```diff
@@ -326,6 +333,39 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     addFeatures(featureList);
   }
 
+  /**
+   * Register the video element to the set of elements currently in use.
+   * @param videoElement the video element to register.
+   * @throws Error - Throws if the element is already used by another player instance.
+   */
+  private static _priv_registerVideoElement(videoElement: HTMLMediaElement) {
+    const errorMessage =
```

Comment:
Why declaring the `errorMessage` variable outside the if?

It would make more sense when reading if it comes after the related check, no?

Reply:
OK I changed it